libsolidity/interface/CompilerStack.cpp

```diff
@@ -1216,8 +1216,9 @@ Json CompilerStack::ethdebug(Contract const& _contract, bool _runtime) const
 	solUnimplementedAssert(!isExperimentalSolidity());
 	evmasm::LinkerObject const& object = _runtime ? _contract.runtimeObject : _contract.object;
 	std::shared_ptr<evmasm::Assembly> const& assembly = _runtime ? _contract.evmRuntimeAssembly : _contract.evmAssembly;
+	solAssert(assembly);
```

Comment:
interfaces will throw an assertion here - e.g. with `test/libsolidity/semanticTests/interfaceID/homer.sol`.. sadly I didn't add a test for interfaces already, thats why you didn't just saw that..

Reply:
do interfaces have an environment (call/create)? that'll be required for the current ethdebug schema

Reply:
hm I think there's nothing extra needed here? like if you compile just an interface, there's no bytecode and thus nothing to debug.

Reply:
oh I just saw
```
======= /Users/alex/git/solidity/test/libsolidity/semanticTests/interfaceID/homer.sol:Simpson =======
IR:

Debug Data (ethdebug/format/program):
null
Debug Data of the runtime part (ethdebug/format/program):
null
```
So its not exactly an empty json.

Reply:
done!